otelcollector/test/ginkgo-e2e/prometheusui/prometheus_ui_test.go

```diff
@@ -93,7 +93,7 @@ var _ = DescribeTable("The Prometheus UI API should return a valid config",
 		Expect(prometheusConfigResult).NotTo(BeNil())
 		Expect(prometheusConfigResult.YAML).NotTo(BeEmpty())
 
-		prometheusConfig, err := config.Load(prometheusConfigResult.YAML, true, nil)
```

Comment:
Hi @bragi92 - we need to up the prom version for config processing tests as well, Looks like these tests are passing since they are using the older version :) https://github.com/Azure/prometheus-collector/blob/5c6ae178f6a3ee4b3d8413b54e629e7bffeb993f/otelcollector/test/ginkgo-e2e/configprocessing/config_processing_test.go#L142

Reply:
Thanks Kaveesh, looks like config_processing_test.go also uses the config.Load() function that will need to be updated to the new parameter format